Microsoft.Toolkit.Uwp.Connectivity/BluetoothLEHelper/ObservableGattCharacteristics.cs

```diff
@@ -110,13 +112,17 @@ public enum DisplayTypes
         /// </summary>
         private string _value;
 
+        private DispatcherQueue _dispatcherQueue;
+
         /// <summary>
         /// Initializes a new instance of the <see cref="ObservableGattCharacteristics"/> class.
         /// </summary>
         /// <param name="characteristic">The characteristic.</param>
         /// <param name="parent">The parent.</param>
         public ObservableGattCharacteristics(GattCharacteristic characteristic, ObservableGattDeviceService parent)
         {
+            _dispatcherQueue = DispatcherQueue.GetForCurrentThread();
```

Comment:
>GetForCurrentThread [](start = 47, length = 19)

What if one of these gets created on a background thread?

Reply:
I've changed them so they can be set after creation as well. We'll call it out on the docs.